wgs/wasm/deliverables/wasm-oci-artifact/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Wasm OCI Artifact layout
+---
+
+# Wasm OCI Artifact layout
+
+This is a guide for an [OCI image format](https://github.com/opencontainers/image-spec/tree/main) for Wasm that can be used across projects in order to enable consistency for users, tooling and registries. 
+This OCI artifact has a specific Wasm `config.mediaType` of `application/vnd.wasm.config.v0+json` to designate it as an [OCI artifact](https://github.com/opencontainers/image-spec/blob/main/artifacts-guidance.md). 
+It contains a representation of Wasm configuration that can be broadly be used across projects (various Wasm runtimes and container runtimes). 
+It includes the ability to include wasm modules or components in the artifact.
+
+It is designed to be compliant with the artifact guidance of the [OCI image spec](https://github.com/opencontainers/image-spec/blob/main/manifest.md#guidelines-for-artifact-usage) 
+in order to be compatible with the widest range of registries today.
+  
+### Goals
+
+- Single OCI compliant artifact type that can be used across container runtimes and Wasm runtime projects (wasmcloud, spin, containerd, podman, etc)
+- Single layer type that is supported where everything is a component, which makes it so all the tools don’t need to know about all possible layer types (eg. static files, Wasm runtime config files, etc.).  
+- Should be able to support an “exploded” representation of the component where each component is a layer and can be combined (“composed”) by either Wasm runtime or container runtime
+- Have fall back if registries don’t support artifacts (via config.media type)
+
+### Non Goals
+
+- Support for use in browsers.  As use cases are presented we may reconsider.
+
+## Manifest format
+
+The Wasm `config.mediaType` configuration allows for easy identification of a Wasm artifact in a registry. See the `config.mediaType` section below for more details on the format of the config file. 
+
+A single layer `mediaType` of `application/wasm` was chosen since a component can be both a library and an executable and so we don't want to need to indicate a difference between components. In addition, a component can always be recomposed into a another type. Although out of scope this does allow for potential browser support in future since browser only supports `application/wasm`.
+
+```json
```

Comment:
Minor nit, I think `jsonc` will give you better syntax highlighting here (JSON with comments).